src/Kunstmaan/DashboardBundle/Command/GoogleAnalyticsDataCollectCommand.php

```diff
@@ -186,7 +186,7 @@ private function getOverviewsOfSegment($segmentId)
         $segmentRepository->initSegment($segment);
 
         // get the overviews
-        return $segment->getOverviews();
+        return $segment->getoverviews();
```

Comment:
I would rather see that the method itself was formatted to a correct camelCase way instead of changing the method call.

Reply:
Good point! I though it would be a bc break to rename the method, but methods are case insensitive, so rename the method and restored these changes!